google/cloud/bigtable/examples/README.md

```diff
@@ -11,6 +11,7 @@ referenced from the Doxygen landing page as "Hello World".
 - [Samples](#samples)
   - [Hello World](#hello-world)
   - [Administer Instances](#administer-instances)
+  - [Administer Tables] (#Table/Column family management)
```

Comment:
Please rename the `Table/Column family management` section to `Administer Tables` and update this link to `#administer-tables`.  That should allow the linking to work correctly